Observer/SendWebhook.php

```diff
@@ -40,8 +40,14 @@ public function execute(\Magento\Framework\Event\Observer $observer)
         $orderData['increment_id'] = $order->getIncrementId();
         $orderData['picqer_magento_key'] = $magentoKey;
 
-        $this->_curl->addHeader("Content-Type", "application/json");
-        $this->_curl->setTimeout(2); // in seconds
+        $this->_curl->setHeaders(
+            'Content-Type', 'application/json'
+        );
```

Comment:
Deze lijkt me niet te kloppen. Dit leest voor mij als 2 losse headers toevoegen.

Als ik [deze](https://github.com/pepe1518/magento2/blob/master/vendor/magento/framework/HTTP/Client/Curl.php#L138) en [deze code](https://github.com/pepe1518/magento2/blob/master/vendor/magento/framework/HTTP/Client/Curl.php#L126) bekijk zou het `->setHeaders(['Content-Type' => 'application/json'])` moeten zijn.

Dat is ook zoals Squeezely de [headers gebruikt](https://github.com/squeezely/magento2-plugin/blob/8c7b728dbea7744b656f33289170f987b7213990/Service/Api/Request.php#L93).

Reply:
Ja dit is niet goed. Dit is ook niet wat ik aan de klant heb voorgesteld.